assets/javascripts/discourse/components/post-voting-comment-actions.js

```diff
@@ -26,4 +56,17 @@ export default class PostVotingCommentActions extends Component {
       },
     });
   }
+
+  @action
+  showFlag() {
+    this.comment.availableFlags = this.comment.available_flags;
+    this.modal.show(FlagModal, {
+      model: {
+        flagTarget: new PostVotingFlag(),
+        flagModel: this.comment,
+        setHidden: () => this.comment.set("hidden", true),
+        site: this.site,
```

Comment:
```suggestion
```

I would import `site` in the modal directly if you need it.

Reply:
I think the reviewables pipeline uses this, tried to import it in the `post-voting-flag` but it didn't worked